exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,4 +1,8 @@
 // Declare your function first
+function divide(num1, num2) {
+    const result = num1 / num2;
```

Comment:
you could skip this line and just have return num1/num2

Reply:
I amended divide function in accordance with your feedback. Thanks Aisha.